port/windows/clock.c

```diff
@@ -60,27 +54,9 @@ oc_clock_time(void)
 oc_clock_time_t
 oc_clock_time_monotonic(void)
 {
-  assert(g_query_perm_frequency != 0);
-
-  LARGE_INTEGER qtime;
-  QueryPerformanceCounter(&qtime); // always succeeds
-  // 10 MHz is a very common QPC frequency on modern PC, so we can simplify the
-  // calculation
-  const LONGLONG tenMHz = 10000000;
-  if (g_query_perm_frequency == tenMHz) {
-    const double multiplier = (double)OC_CLOCK_SECOND / tenMHz;
-    return (LONGLONG)(qtime.QuadPart * multiplier);
-  }
-
-  // Instead of just having "(qtime * OC_CLOCK_SECOND) /
-  // g_query_perm_frequency", the algorithm below prevents overflow when qtime
-  // is sufficiently large.
-  const LONGLONG whole =
-    (qtime.QuadPart / g_query_perm_frequency) * OC_CLOCK_SECOND;
-  const LONGLONG part =
-    (LONGLONG)((qtime.QuadPart % g_query_perm_frequency) *
-               ((double)OC_CLOCK_SECOND / g_query_perm_frequency));
-  return whole + part;
+  // The return value is the number of milliseconds that have elapsed since the
+  // system was started.
+  return ((oc_clock_time_t)GetTickCount64()) * 1000;
```

Comment:
Why not use the more precise `QueryPerformanceCounter`. Also you need to do conversion to oc_clock_time_t

Reply:
Note that the QueryPerformanceCounter() function is hardware-dependent and may not be available on all systems. However, it is generally available on modern Windows systems and provides a high-resolution timer that is suitable for many timing applications.